### PR TITLE
added export api utilization for task document + annotations

### LINKED ISSUE #12: Change task annotation and document to use export api
In task - we should use export api to retrieve all documents and annotations together in order to prevent hundreds of request to server.

### PR DESCRIPTION
**Pull request by nikhiln**
Fixed #12

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 numpy>=1.14.3
 pytest>=3.6.1
-requests>=2.20.0
\ No newline at end of file
+requests>=2.20.0
+pandas>=0.23.4
```
Comment (zermelozf) [inline requirements.txt]: @nikhiln could we achieve the same without needing pandas? I'd like to keep dependencies to a minimum if possible.

File: linalgo/client.py [hunk 2]
```diff
@@ -81,6 +87,19 @@ def request(self, url, query_params={}):
...
+        headers = {'Authorization': f"Token {self.access_token}"}
+        # stream the file
+        with closing(requests.get(url, stream=True, headers=headers, params=query_params)) as res:
+            if res.status_code == 401:
+                raise Exception(f"Authentication failed. Please check your token.")
+            if res.status_code == 404:
+                raise Exception(f"{url} not found.")
+            elif res.status_code != 200:
+                raise Exception(f"Request returned status {res.status_code}")
+            df = pd.read_csv(io.BytesIO(res.content), compression='zip', header=0, sep=',', quotechar='"')
```
Comment (zermelozf) [inline linalgo/client.py]: @nikhiln I think we could just use the csv standard lib here right?

**Review by zermelozf — CHANGES_REQUESTED** [review]
@nikhiln Could you change the csv reader to the standard one so we can remove the pandas dependency please?

File: requirements.txt (hunk 1 above)
Comment (nikhiln) [reply requirements.txt]: @zermelozf We can, but we would need to then extract csv and parse it. Pandas does that conveniently. Aren't we using the pandas for data science part later on?

File: linalgo/client.py (hunk 2 above)
Comment (nikhiln) [reply linalgo/client.py]: @zermelozf Yes, replaced it with standard csv lib.

File: requirements.txt (hunk 1 above)
Comment (nikhiln) [reply requirements.txt]: @zermelozf Replaced it with standard csv lib.

**Review by zermelozf — APPROVED** [review]
+1

**Pull request merged**